plugins/workbench-resources/src/connect.ts

```diff
@@ -83,13 +83,13 @@ export async function connect (title: string): Promise<Client | undefined> {
 
   setMetadata(presentation.metadata.Token, token)
 
-  if (workspaceLoginInfo?.creating === true) {
+  if (['pending-creation', 'creating'].includes(workspaceLoginInfo?.mode ?? '')) {
```

Comment:
Could we have some predicate instead? It used 3 times.

Reply:
Sure, will extract it into an utility in the next PR.